### PR TITLE
CIV-12079 confirmation pages are not right when case moves to in mediation

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-12079

### Change description ###

Confirmation pages are not right when case moves to InMediation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -1061,7 +1061,8 @@
         "YES_MEDIATION": {
           "WHAT_HAPPENS_NEXT_TEXT_PARA_1": "You agreed to try free mediation.",
           "WHAT_HAPPENS_NEXT_TEXT_PARA_2": "Your mediation appointment will be arranged within 28 days."
-        }
+        },
+        "DOWNLOAD_YOUR_HEARING_REQUIREMENTS": "Download your hearing requirements(PDF)"
```
[the remark this answers is not in the document]
Comment (jeswanth-hmcts) [reply src/main/modules/i18n/locales/en.json]: amend as per the alphabetical order  within the object

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -1061,7 +1061,8 @@
         "YES_MEDIATION": {
           "WHAT_HAPPENS_NEXT_TEXT_PARA_1": "You agreed to try free mediation.",
           "WHAT_HAPPENS_NEXT_TEXT_PARA_2": "Your mediation appointment will be arranged within 28 days."
-        }
+        },
+        "DOWNLOAD_YOUR_HEARING_REQUIREMENTS": "Download your hearing requirements(PDF)"
```
[the remark this answers is not in the document]
Comment (jeswanth-hmcts) [reply src/main/modules/i18n/locales/cy.json]: amend as per the alphabetical order  within the object

**Review by neeta-hmcts — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -1364,7 +1364,7 @@
         "REQUESTED_MORE_TIME_TO_RESPOND": "Mae {{defendantName}} wedi gofyn am fwy o amser i ymateb.",
         "RESPONSE_BY_POST": "Bydd yr hawliad yn parhau drwy'r post",
         "WAITING_COURT_REVIEW": "Aros i'r llys adolygu'r achos",
-        "BOTH_SIGNED_SETTLEMENT_AGREEMENT": "You’ve both signed a settlement agreement.",       
+        "BOTH_SIGNED_SETTLEMENT_AGREEMENT": "You’ve both signed a settlement agreement.",
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:1367]: Same text and we have the welsh translation.

"SETTLEMENT_SIGNED": "Mae'r ddau ohonoch wedi llofnodi cytundeb setlo.",

Comment (kenneth-hmcts) [reply src/main/modules/i18n/locales/cy.json]: Updated from review

**Review by laarmada — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**